.github/workflows/extremes.yml

```diff
@@ -23,12 +23,12 @@ jobs:
         with:
           enable-cache: "true"
           cache-suffix: ${{ matrix.python-version }}
-          cache-dependency-glob: "**requirements*.txt"
+          cache-dependency-glob: "pyproject.toml"
       - name: install-minimum-versions
         run: uv pip install tox virtualenv setuptools pandas==0.25.3 polars==0.20.3 numpy==1.17.5 pyarrow==11.0.0 scipy==1.5.0 scikit-learn==1.1.0 tzdata --system
       - name: install-reqs
         run: |
-          uv pip install -r requirements-dev.txt --system
+          uv pip install -e ".[dev]" --system
```

Comment:
Somehow this is uninstalling pyarrow 11 and installs pyarrow v17.0.0.

I can see this happening consistently in the most recent PRs

Reply:
ah well-spotted

we should probably have `[backends]` as well as `[dev]`

Reply:
though i still don't see why that would cause this to install pyarrow 17 🤔

Reply:
looking here it' still pyarrow 11 https://github.com/narwhals-dev/narwhals/actions/runs/12380396751/job/34556674164

Reply:
ooh but not here, you're right! https://github.com/narwhals-dev/narwhals/actions/runs/12394099025/job/34596775668?pr=1613

Reply:
True, but both:
- https://github.com/narwhals-dev/narwhals/actions/runs/12394099025/job/34596775668?pr=1613
- https://github.com/narwhals-dev/narwhals/actions/runs/12393431408/job/34594660478?pr=1614

end up with pyarrow 17

Reply:
i think it's the new dask release

Reply:
found it, it's pyarrow-stubs! thanks again for spotting this, legend